### ISSUE TITLE
Error when running the convert_plan.py script

### ISSUE THREAD

**Issue by harsh-agar** [NVIDIA-Jetson/tf_to_trt_image_classification#23]
Hi,

To run the script convert_plan.py, I use the command 

`python scripts/convert_plan.py data/frozen_graphs/mobilenet_v1_1p0_224.pb data/plans/mobilenet_v1_1p0_224.plan input 224 224 MobilenetV1/Logits/SpatialSqueeze 1 0 float
`

The script ends with an error :

After conv-act fusion: 115 layers
After tensor merging: 115 layers
After concat removal: 115 layers
cudnnEngine.cpp (56) - Cuda Error in initializeCommonContext: 4
cudnnEngine.cpp (56) - Cuda Error in initializeCommonContext: 4

This is when I run it on Ubuntu 16.04 with Nvidia 1080ti with Tensorflow 1.4, TensorRT 3.0.4 and Cuda 8.
Are there any changes which I need to do in the Make file for this to work?

**Comment by harsh-agar** [comment]
Got it working!! 
Hence closing.

**Comment by yw2278** [comment]
> Got it working!!
> Hence closing.

how do you solve the problem? May I ask?
